jobs/README.md

```diff
@@ -4,6 +4,19 @@ In this section you can find some job openings listed by our community members.
 
 __Be advised that Job Openings older than 3 months will be removed for the sake of keeping current. Feel free to repost.__
 
+## Βlockchain / Smart contract Engineer (Remote)
+
+Good evening!
+
+I ([maninak](https://app.slack.com/client/T06RGB4BU/C06RGB4DQ/user_profile/U2F3S7MMW)) am forwarding a job listing for a friend and colleague based in London. From what I understand it's about a promising new project in the crypto space, for which an international team is being set up and with some big names in the industry behind it, therefore it's a bit hush-hush in the beginning, hence the brief listing.
```

Comment:
I am not sure if your slack is of any use within a job add. Maybe if you are a part of that team it could help others reach you out. But even thought they could reach you out through github!